src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,24 @@
-export const PostInfo = () => (
-  <>Put the post here</>
-);
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+
+import './PostInfo.scss';
+
+export const PostInfo = ({ post }) => {
+  const { title, body, user, comments } = post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{title}</h3>
+        {user && <UserInfo user={user} />}
+      </div>
+      <p className="PostInfo__body">{body}</p>
+      <hr />
```

Comment:
don't forget about code style, add a line between inner elements for better readability (check all files)